client/src/Views/Landing/Login.js

```diff
@@ -3,69 +3,105 @@ import { Grid, withStyles, Typography, TextField } from "@material-ui/core";
+    // to prompt them to do it
+    this.setState({
+      gmailDialogOpen: true,
+    });
+  };
+
+  handleClose = () => {
+    this.setState({
+      gmailDialogOpen: false,
     });
   };
 
   render() {
     return (
-      <Grid
-        item
-        container
-        direction="column"
-        alignContent="center"
-        alignItems="center"
-        spacing={7}
-        className={this.props.classes.root}
-      >
-        <Grid item>
-          <Typography className={this.props.classes.header}>Login</Typography>
-        </Grid>
-
+      <span>
         <Grid
           item
           container
           direction="column"
           alignContent="center"
-          spacing={2}
+          alignItems="center"
+          spacing={7}
+          className={this.props.classes.root}
         >
-          <Grid item className={this.props.classes.input}>
-            <TextField
-              label="Your email"
-              name="email"
-              type="email"
-              variant="outlined"
-              fullWidth
-              onChange={this.onChange}
-              value={this.state.email}
-            />
+          <Grid item>
+            <Typography className={this.props.classes.header}>Login</Typography>
           </Grid>
-          <Grid item className={this.props.classes.input}>
-            <TextField
-              label="Password"
-              name="password"
-              type="password"
-              variant="outlined"
-              fullWidth
-              onChange={this.onChange}
-              value={this.state.password}
-            />
+
+          <Grid
+            item
+            container
+            direction="column"
+            alignContent="center"
+            spacing={2}
+          >
+            <Grid item className={this.props.classes.input}>
+              <TextField
+                label="Your email"
+                name="email"
+                type="email"
+                variant="outlined"
+                fullWidth
+                onChange={this.onChange}
+                value={this.state.email}
+              />
+            </Grid>
+            <Grid item className={this.props.classes.input}>
+              <TextField
+                label="Password"
+                name="password"
+                type="password"
+                variant="outlined"
+                fullWidth
+                onChange={this.onChange}
+                value={this.state.password}
+              />
+            </Grid>
           </Grid>
-        </Grid>
 
-        <Grid item>
-          <StyledButton>Login</StyledButton>
+          <Grid item>
+            <StyledButton onClick={this.onClick}>Login</StyledButton>
+          </Grid>
+          <GmailDialog
+            open={this.state.gmailDialogOpen}
+            onClose={this.handleClose}
+            endRoute="/login"
+          />
         </Grid>
-      </Grid>
+      </span>
```

Comment:
You've got the right idea, but I meant that the `<span>` wrapper isn't necessary

Reply:
Doh, I forgot to delete the span. Good spot!